setup.py

```diff
@@ -3,6 +3,10 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+extras_require = {
```

Comment:
@jeff-hernandez lets add an install workflow and test that the update checker installs properly.
https://github.com/alteryx/featuretools/blob/main/.github/workflows/install_test.yml#L34

Reply:
Added workflow